dataplane/src/pipeline/pipeline.rs

```diff
@@ -148,5 +148,14 @@ mod test {
             (p2_ttl as usize) - DynStageGenerator::num_ttl_decs(num_stages),
             p2_out.try_ipv4().unwrap().ttl() as usize
         );
+
+        // Check try_ip() and try_ipv4() are consistent
+        let p1_ipv4 = p1_out.try_ipv4().expect("Expected IPv4 packet");
```

Comment:
I'm not a fan of testing this here instead of net/src/headers.rs but not worth holding things up over it.

Reply:
Yep, I thought about adding tests to header.rs but having something just for `TryIp` felt weird. Adding for all other traits felt beyond the scope. We can probably refactor these at some point. Thanks!